flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/TestChangeLogTable.java

```diff
@@ -24,6 +24,7 @@
 import java.util.List;
 import org.apache.flink.types.Row;
 import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.AssertHelpers;
```

Comment:
why is the unit test change not ported to 1.13?

Reply:
I'm not sure whether we should keep developing on the latest version, so I was hesitant to add changes to the 1.13 version, not to mention we are developing 1.15 now. But there are some production CDC cases still base on Flink 1.13, let me port them anyway.

Reply:
I saw you ported the main src to 1.13 and not the unit test. Hence I was asking. I am fine if we don't update 1.13 at all.

Reply:
I'd be fine with not adding for 1.13 in this PR (and backporting it once review is done).